test.py

```diff
@@ -16,12 +19,21 @@ def test(rank, args, T, shared_model):
   model = ActorCritic(env.observation_space, env.action_space, args.hidden_size)
   model.eval()
 
+  if not os.path.exists('results'):
```

Comment:
This should not be needed since it is done in the main file before testing is launched.